packages/client/src/App.vue

```diff
@@ -1,3 +1,8 @@
+<script setup lang="ts">
+import Header from './components/Header.vue';
```

Comment:
should use the alias `@` for consistency
may add other aliases in the future if needed

Reply:
```suggestion
import Header from '@/components/Header.vue';
```